zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/IPAuthenticationProvider.java

```diff
@@ -112,11 +253,13 @@ public boolean matches(String id, String aclExpr) {
         mask(aclAddr, bits);
         byte[] remoteAddr = addr2Bytes(id);
         if (remoteAddr == null) {
+            LOG.info("Address is null");
```

Comment:
I have the impression that you added these log messages for debugging purposes, in which case please remove them.